recipes/enso_metrics/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "enso_metrics" %}
+{% set version = "v1.1.0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/CLIVAR-PRP/ENSO_metrics/archive/refs/tags/{{ version }}.tar.gz
+    sha256: e933f373be1d793b801ed7155690fc4759a463da5931f18b5117ba9b259840b3    
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - cdms2
+    - cdutil
+    - genutil
+    - numpy
+    - scipy
+    - udunits2
+
+test:
+  imports:
+    - EnsoMetrics.EnsoCollectionsLib
+    - EnsoMetrics.EnsoComputeMetricsLib
```

```suggestion
test:
  imports:
    - EnsoMetrics.EnsoCollectionsLib
    - EnsoMetrics.EnsoComputeMetricsLib
  commands:
    - pip check
  requires
    - pip
```

This also checks the deps.